src/sentry/api/bases/organization.py

```diff
@@ -284,56 +284,63 @@ def get_projects(
         standardize how this is used and remove this parameter.
         :param project_ids: Projects if they were passed via request
         data instead of get params
+        :param project_slugs: Project slugs if they were passed via request
+        data instead of get params
         :return: A list of Project objects, or raises PermissionDenied.
+
+        NOTE: If both project_ids and project_slugs are passed, we will default
+        to fetching projects via project_id list.
         """
-        if project_ids is None:
+        qs = Project.objects.filter(organization=organization, status=ObjectStatus.ACTIVE)
+        if project_ids is not None:
+            if project_ids == ALL_ACCESS_PROJECTS:
+                # All projects i have access to
+                include_all_accessible = True
+            elif project_ids:
+                qs.filter(id__in=project_ids)
+            # No project ids === `all projects i am a member of`
+        else:
             slugs = project_slugs or set(filter(None, request.GET.getlist("projectSlug")))
             if ALL_ACCESS_PROJECTS_SLUG in slugs:
-                project_ids = ALL_ACCESS_PROJECTS
+                # All projects i have access to
+                include_all_accessible = True
             elif slugs:
-                projects = Project.objects.filter(
-                    organization=organization, slug__in=slugs
-                ).values_list("id", flat=True)
-                project_ids = set(projects)
-
-                # return early to prevent passing empty set of project_ids to _get_projects_by_id
-                # which would return all projects in the organization
-                if not project_ids:
```

Comment:
@cathteng we're no longer returning if we don't have any project_ids from slugs (eg. pass an empty list of slugs)

This means you can still query for all projects I am a member of via project_slug too - which i don't know if we do anywhere...

Any idea what this might affect? (or why we initially made this change int he first place?)

Reply:
Actually - i see the [test in the original pr](https://github.com/getsentry/sentry/pull/39997/files#diff-bdf9ad505538102dca1db93d666ac96850c4692da3e62a70ca7a7a2e0f7534efR392)

This makes sense - 
If the query results does not include the projects we requested, then we shouldn't raise a `PermissionDenied`, we should simply return the list sans missing projects 🤔 

I'll make this change